components/var_dumper/introduction.rst

```diff
@@ -62,6 +62,17 @@ current PHP SAPI:
     You'll also learn how to change the format or redirect the output to
     wherever you want.
 
+.. tip::
+
+    In order to have the ``dump()`` function always available, you can install
+    it globally in your dev environment:
```

Comment:
The phrase `... in your dev environment`  may sound confusing. This has nothing to do with the `dev` environment of Symfony applications. I would replace it by:

```
In order to have the ``dump()`` function available for any PHP script or application and not only for your Symfony applications, you can install it globally in your machine:
```